rpc-client-api/src/response.rs

```diff
@@ -1,3 +1,6 @@
+pub use solana_transaction_status::{
+    UiReturnDataEncoding as ReturnDataEncoding, UiTransactionReturnData as RpcTransactionReturnData,
+};
```

Comment:
Considering this is a breaking change, how about breaking the current consumers loudly on build by not doing these re-exports? Would that still work with the backwards compatibility policy for unstable APIs?

Reply:
This'll just break consumers of return data from simulate. Yeah, works for me.